draft-ietf-tsvwg-rfc4960-bis.xml

```diff
@@ -1242,7 +1243,7 @@ to the INIT chunk.</t>
 <t>The Chunk Flags field in INIT chunks is reserved, and all bits in it SHOULD
 be set to 0 by the sender and ignored by the receiver.
 The sequence of parameters within an INIT chunk can be processed in any
-order.</t>
+order (but must appear within the chunk in the order indicated above).</t>
```

Comment:
I actually would suggest to remove the sentence
```
The sequence of parameters within an INIT chunk can be processed in any order.
```
since the newly added error handling sort of implies that you do the checks in sequence (or at least check for initiate_tag == 0 first). 
So I would vote for:
* Remove the sentence cited above.
* Add the sentence about the Chunk flags also to the INIT-ACK chunk.

This keeps things a simple and as consistent as possible between the INIT and INIT ACK chunk. OK?

Reply:
The next revision will contain the changes suggested by me above. I'm resolving the conversation. If you don't agree, please unresolve the conversation.